dungeon/src/core/Game.java

```diff
@@ -416,10 +417,10 @@ public static ILevel currentLevel() {
    * <p>{@link Point#toCoordinate} will be used, to convert the point into a coordinate.
    *
    * @param point Point from where to get the tile
-   * @return the tile at the given point.
+   * @return an Optional containing the tile, or empty if not found
    */
-  public static Tile tileAT(final Point point) {
-    return currentLevel().tileAt(point);
+  public static Optional<Tile> tileAT(final Point point) {
+    return Optional.ofNullable(currentLevel().tileAt(point));
```

Comment:
Du musst hier tiefer gehen.
Die Funktion ruft ja nur currentLevel().tileAt(point) auf.
Du musst den ursprung des `return null` finden und dort das Optional einbauen, dann ziehst du das ganze nach vorne durch. 

Mach das gerne in einen seperaten PR und dann greifst du diesen PR wieder auf